stack_master.gemspec

```diff
@@ -39,7 +39,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "timecop"
   spec.add_dependency "os"
   spec.add_dependency "ruby-progressbar"
-  spec.add_dependency "commander"
+  spec.add_dependency "commander", "<= 4.4.5"
```

Comment:
Maybe `~> 4.4` would be better than anything less than this specific version?

Reply:
Sorry I didn't specify in this PR, the issue was introduced in `4.4.6`

Reply:
Ah yeah we can do something like `  spec.add_dependency "commander", "~> 4.4", "<= 4.4.5"`